proto/event.proto

```diff
@@ -23,7 +23,6 @@ message Value {
     double float = 5;
     bool boolean = 6;
   }
-  bool explicit = 3;
```

Comment:
There's an argument for keeping this around, either commented out or renamed to something like `DEPRECATED`. That way we'll see that tag `3` was used and can continue to make non-breaking changes.

Reply:
This can be `bool explicit = 3 [deprecated=true];`, or we could mark it as a [`reserved`](https://developers.google.com/protocol-buffers/docs/proto3#enum_reserved) value.

Reply:
I like using `reserved` for this.

Reply:
`reserved` sounds perfect!